mesonbuild/build.py

```diff
@@ -2402,6 +2414,15 @@ def is_linkable_target(self):
         if suf == '.a' or suf == '.dll' or suf == '.lib' or suf == '.so':
             return True
 
+    def should_install(self) -> bool:
+        return self.target.should_install()
+
+    def is_internal(self) -> bool:
+        return self.target.is_internal()
```

Comment:
Hmm, looking at the code, `t.is_internal()` is never called on a CustomTarget or CustomTargetIndex, why do we implement this method? Same above.